flows/runs/legacy.go

```diff
@@ -96,24 +95,3 @@ func arrayToObject(array *types.XArray) *types.XObject {
 	}
 	return types.NewXObject(properties)
 }
-
-// finds the last webhook response that was saved as extra on a result
-func lastWebhookSavedAsExtra(r *run) types.XValue {
```

Comment:
this is a potentially backwards compat breaking change if someone is relying on small webhook responses being available between sprints... but I think it's worth ripping off the bandaid because 1) we've always told users that `@webhook` isn't persisted between sprints and 2) it's unpredictable in that if the response goes over 10,000 chars, this isn't set.

Reply:
So they need to save whatever is needed from the webhook in a result variables so they can use those later, right?